testhelper/destination/kafka.go

```diff
@@ -0,0 +1,104 @@
+package main_test
+
+import (
+	_ "encoding/json"
+	"fmt"
+	_ "github.com/Shopify/sarama"
+	_ "github.com/lib/pq"
+	"github.com/ory/dockertest"
+	dc "github.com/ory/dockertest/docker"
+	"github.com/phayes/freeport"
+	"log"
+	"strconv"
+)
+
+type Test struct {
+	pool             *dockertest.Pool
+	err              error
+	network          *dc.Network
+	brokerPort       string
+	localhostPort    string
+	localhostPortInt int
+}
+
+func SetupKafka(pool *dockertest.Pool) (*Test, *dockertest.Resource) {
+	Test := &Test{}
+	Test.pool = pool
+	fmt.Println("Set zookeper")
+	Test.network, Test.err = Test.pool.Client.CreateNetwork(dc.CreateNetworkOptions{Name: "kafka_network"})
+	if Test.err != nil {
+		log.Printf("Could not create docker network: %s", Test.err)
+	}
+	zookeeperPortInt, err := freeport.GetFreePort()
+	if err != nil {
+		fmt.Println(err)
+	}
+	zookeeperPort := fmt.Sprintf("%s/tcp", strconv.Itoa(zookeeperPortInt))
+	zookeeperclientPort := fmt.Sprintf("ZOOKEEPER_CLIENT_PORT=%s", strconv.Itoa(zookeeperPortInt))
+	log.Println("zookeeper Port:", zookeeperPort)
+	log.Println("zookeeper client Port :", zookeeperclientPort)
+
+	z, err := Test.pool.RunWithOptions(&dockertest.RunOptions{
+		Repository: "confluentinc/cp-zookeeper",
+		Tag:        "latest",
+		NetworkID:  Test.network.ID,
+		Hostname:   "zookeeper",
+		PortBindings: map[dc.Port][]dc.PortBinding{
+			"2181/tcp": {{HostIP: "zookeeper", HostPort: zookeeperPort}},
+		},
+		Env: []string{"ZOOKEEPER_CLIENT_PORT=2181"},
+	})
+	if err != nil {
+		fmt.Println(err)
+	}
+	return Test, z
+}
+
+func SetKafka(Test *Test, z *dockertest.Resource) *dockertest.Resource {
```

Comment:
this can remain unexported
```suggestion
func setKafka(Test *Test, z *dockertest.Resource) *dockertest.Resource {
```